evalml/tests/data_checks_tests/test_data_check_action.py

```diff
@@ -158,3 +158,33 @@ def test_convert_dict_to_action(dummy_data_check_name):
         data_check_action_dict_with_other_metadata
     )
     assert data_check_action == expected_data_check_action
+
+
+@pytest.mark.parametrize(
+    "action_code,expected_code",
+    [
+        ("drop_rows", DataCheckActionCode.DROP_ROWS),
+        ("Drop_col", DataCheckActionCode.DROP_COL),
+        ("TRANSFORM_TARGET", DataCheckActionCode.TRANSFORM_TARGET),
+    ],
+)
+def test_data_check_action_equality_string_input(
```

Comment:
nitpick but maybe we want to update the names `action_code` and `expected_code`? Seems like what we're trying to test is that the stringified version is equivalent to the enum, so something like `action_code_string` vs 'action_code`?